man/man8/zpool.8

```diff
@@ -2727,6 +2731,10 @@ them on \fBzpool create\fR or \fBzpool add\fR by setting ZFS_VDEV_DEVID_OPT_OUT.
 .B "ZPOOL_SCRIPTS_AS_ROOT"
 Allow a privilaged user to run the \fBzpool status/iostat\fR with the \fB-c\fR
 option.  Normally, only unprivilaged users are allowed to run \fB-c\fR.
+.TP
+.B "ZPOOL_SCRIPTS_PATH"
+The search path for scripts when running \fBzpool status/iostat\fR with the \fB-c\fR
+option. This is a colon-separated list of directories.
```

Comment:
May also want to add that ZPOOL_SCRIPTS_PATH overrides the default \<etc\>/zfs/zpool.d and \<home\>/.zpool.d script paths.